aya/src/maps/mod.rs

```diff
@@ -582,11 +597,13 @@ impl MapData {
     /// This API is intended for cases where you have received a valid BPF FD from some other means.
     /// For example, you received an FD over Unix Domain Socket.
     pub fn from_fd(fd: OwnedFd) -> Result<Self, MapError> {
+        // TODO(https://github.com/aya-rs/aya/issues/612): should this function take a MapFd?
+        let fd = MapFd(fd);
```

Comment:
nit: probably would read better from a domain POV to only wrap it in `MapFd` after confirming that it's a `MapFd` (i.e., after `bpf_map_get_info_by_fd` succeeds).

Reply:
Done.